File: mysql.rb
require 'rubygems'
require 'mysql.bundle'

puts Mysql::VERSION
#my = Mysql.new(hostname, username, password, databasename)
con = Mysql.new('FIXME_HOST', 'FIXME_USER', 'FIXME_PASS', 'FIXME_DB')
rs = con.query('select * from `domains_list` where `domainname` not like "%arpa" limit 0, 5')
rs.each_hash do |h|
   puts h['domainname']
end
con.close

File: mysql.py
import _mysql
db=_mysql.connect("FIXME_HOST","FIXME_USER","FIXME_PASS","FIXME_DB")
print db.info()
r = db.query('select domainname from domains_list limit 0, 10 ')
# r=db.store_result()
for o in r.fetch_row():
	print o[0]
	

print 'hello'